Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -80,13 +86,36 @@ public final class ProjectPamphletViewModel: ProjectPamphletViewModelType, Proje
     let freshProjectAndRefTag = freshProjectAndRefTagEvent.values()
 
     let ctaButtonTapped = freshProjectAndRefTag
-      .takeWhen(self.backThisProjectTappedProperty.signal)
-      .map { project, refTag in
-        (project, refTag)
+      .takePairWhen(self.pledgeCTAButtonTappedProperty.signal)
+      .map(unpack)
+
+    let goToManagePledge = ctaButtonTapped
+      .filter(canShowManageViewPledgeScreen(_:_:state:))
+      .map { (project, refTag, _) -> PledgeData in
+        let r = reward(from: project.personalization.backing, inProject: project)
+        return PledgeData(project: project, reward: r, refTag: refTag)
+      }
+
+    self.goToManageViewPledge = goToManagePledge
+      .filter { _ in featureNativeCheckoutPledgeViewIsEnabled() }
+
+    self.goToDeprecatedManagePledge = ctaButtonTapped
+      .filter(shouldGoToDeprecatedManagePledge(_:_:state:))
```

Comment:
Same thing with `refTag`...is this because we'll need it in the future but don't need it just now?

Reply:
Yeah, I'd probably try to carry over as little unused things as possible..this makes it easier to reason about code and will be less confusing in the future (read in a week when we forget WTF was this) :D